tests/cross/src/test/scala/tests/pc/PcDefinitionSuite.scala

```diff
@@ -161,7 +199,15 @@ class PcDefinitionSuite extends BasePcDefinitionSuite {
        |import scala.concurrent./*scala/concurrent/Future. Future.scala*/@@Future
        |object Main {
        |}
-       |""".stripMargin
+       |""".stripMargin,
+    compat = Map(
+      "3.0" ->
+        """|
+           |import scala.concurrent./*scala/concurrent/Future. Future.scala*//*scala/concurrent/Future# Future.scala*/@@Future
```

Comment:
Why do we get both object and class here? Or more persaisly, why we didn't get it in Scala 2?

Reply:
I thought to check if we can do the same for scala2 but decided to save time and keep PR only about scala3.
I don't think that's critical and it works as it was initially implemented in `Interactive`